policies/api/http/transport.go

```diff
@@ -42,21 +43,31 @@ func MakeHandler(tracer opentracing.Tracer, svcName string, svc policies.Service
 	}
 	r := bone.New()
 
-	r.Post("/policies/agent", kithttp.NewServer(
+	r.Post("/policies/agents", kithttp.NewServer(
```

Comment:
this should stay `agent` singular. sorry I think you must have looked at the architecture diagram, which erroneously does say `agents`, we need to fix that. our openapi though is singular, and the text on the architecture page is singular.